tutoroars/templates/oars/jobs/init/clickhouse-demo-xapi-data.sh

```diff
@@ -9,14 +9,18 @@ case "{{ OARS_CLICKHOUSE_LOAD_DEMO_DATA }}" in
         echo "Loading demo xAPI data..."
         pip install git+https://github.com/openedx/xapi-db-load@0.1#egg=xapi-db-load==0.1
         pip install pandas  # clickhouse_connect is missing this
-        xapi-db-load --backend clickhouse \
+        xapi-db-load --backend ralph_clickhouse \
```

Comment:
It's interesting to me that this is where you ran into issues from a fresh build because I'm getting them on the superset admin user not existing so not even getting this far. I'll beat my head on this some more today.

Reply:
I have made basically 0 progress today on this because I can't get past my Superset error, which is actually complaining that the xapi table doesn't exist even though it does. I believe it's a timeout issue connecting to CH, but I'm getting so many indent errors on the shell script and each iteration takes ~30 mins. I'm taking the rest of today as a sick day since I've been sick the past couple of days, and will come back to all of this tomorrow.

Reply:
Blergh.. sorry you're feeling ill @bmtcril ! I'll have another look today with the updated Tutor, and push anything I can. We'll get through this! 😄

Reply:
@bmtcril I just watched events get piped from the event routing backends to clickhouse and through to the Superset dashboard! It was so exciting!!

Reply:
This has been resolved (for me at least) by https://github.com/openedx/tutor-contrib-ralph/pull/4/commits/a438496f87fcc08cffd494255bbb3d85283d0175 which did:

> 2. (Bad practice, easy, would have to be temporary) Run Ralph with CLICKHOUSE_ADMIN_USER instead of OARS_CLICKHOUSE_LRS_USER

Not the best answer, but I couldn't get any of the others to work, so..